_includes/parse-server/file-adapters.md

```diff
@@ -175,7 +175,67 @@ var api = new ParseServer({
 
 Don't forget to change **S3_ACCESS_KEY**, **S3_SECRET_KEY** and **S3_BUCKET** to their correct value.
 
-##### S3Adapter constructor options
+### Digital Ocean Spaces
+
+Spaces is an S3-compatible provided by Digital Ocean. It's use the same api as S3 so you can use it with the S3 Adapter.
+You just need to change the AWS Endpoint to point to your Spaces endpoint.
+
+```javascript
+const s3Options = {
+  bucket: "SPACES_BUCKET_NAME",
+  baseUrl: "SPACES_BASE_URL",
+  region: "SPACES_REGION",
+  directAccess: true,
+  globalCacheControl: "public, max-age=31536000",
+  bucketPrefix: "SPACES_BUCKET_PREFIX",
+  s3overrides: {
+    accessKeyId: "SPACES_ACCESS_KEY",
+    secretAccessKey: "SPACES_SECRET_KEY",
+    endpoint: 'SPACES_ENDPOINT'
+  }
+};
+```
+
+### Linode Object Storage
+
+Object Storage is an S3-compatible storage service from Linode. We can configure our S3Adapter to use Linode's service. Please refer to [this guide](https://www.linode.com/docs/guides/how-to-use-object-storage/) for more details on Linode's API.
+
+```js
+const s3Options = {
+  bucket: "S3_BUCKET_NAME",
+  baseUrl: "S3_BASE_URL", // https://myBucket.myRegion.linodeobjects.com
+  region: "S3_REGION", // possible values: eu-central-1 or us-east-1
+  directAccess: false,
+  s3overrides: {
+    accessKeyId: "S3_ACCESS_KEY", // bucket access key
+    secretAccessKey: "S3_SECRET_KEY", // bucket secret key
+    endpoint: "S3_ENDPOINT", // regionName.linodeobjects.com
+  },
+};
+```
+
+### Backblaze
```

Comment:
I'm not sure that is the correct heading level? How can `Backblaze` (and the other 2) have a higher level than `S3Adapter constructor options`? From this is looks like `S3Adapter constructor options` was a sub of  `Backblaze`.

Reply:
You are right. Missed that. I like to keep the s3 providers' section as it is now. What if I lift `constructor options` up one more level? It will stay at the same level as others. Or do you have any other suggestions? I can implement that, too.

Reply:
![image](https://user-images.githubusercontent.com/1928541/117195472-fb4e5000-aded-11eb-9101-8365853f924e.png)

Reply:
Shouldn't the s3 compatible headlines be subs of the Adapter Option headline? Now that we reduced that to the options object, these are really only options examples.

Reply:
Then I suggest to add the headlines like this:
- Adapter Options
- S3-compatible Services
  - Digital Ocean Spaces
  - ...

Reply:
Done and pushed. Can you please check it? Thanks